### PR TITLE
AD-1729: Update ETL for DS -> C2M2 (Y4Q3)

### PR DESCRIPTION
**Pull request by wnkhan**

### CODE REVIEW

File: kf_to_c2m2_etl/kf_table_combiner.py [hunk 1]
```diff
@@ -83,10 +83,13 @@ def add_tables_to_df_dict(self):
                 file_path = os.path.join(file_locations.get_ingested_path(),f'{table_to_endpoint_name[table_name]}.csv')
 
                 if table_name in kf_tablenames:
-                    if table_name in kf_tables_with_visibility and is_column_present(file_path, 'visible'):
-                        table_df = pd.read_csv(file_path, low_memory=False).query('visible == True')
-                    else:
-                        table_df = pd.read_csv(file_path, low_memory=False)
+                    table_df = pd.read_csv(file_path, low_memory=False)
+
+                    if table_name == "genomic_files":
```
Comment (haynescd) [inline kf_to_c2m2_etl/kf_table_combiner.py:88]: Looks good, maybe just a comment for future william/me about why we filtered these parquet files.

**Review by haynescd — APPROVED** [review]
LGTM

File: kf_to_c2m2_etl/kf_table_combiner.py (hunk 1 above)
Comment (wnkhan) [reply kf_to_c2m2_etl/kf_table_combiner.py:88]: Good idea. I'll add it. Thanks for the review.

**Review by Christina-J-Diaz — APPROVED** [review]
looks good, ty!

**Pull request merged**